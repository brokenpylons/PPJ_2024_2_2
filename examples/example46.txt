(10.5 * (m^2 - 3) / (n^(1/3))) + (p + 2 * q)^2 / (1.5^4)
