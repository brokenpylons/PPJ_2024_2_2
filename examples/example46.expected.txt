lparen("(") real("10.5") times("*") lparen("(") variable("m") pow("^") real("2") minus("-") real("3") rparen(")") divides("/") lparen("(") variable("n") pow("^") lparen("(") real("1") divides("/") real("3") rparen(")") rparen(")") rparen(")") plus("+") lparen("(") variable("p") plus("+") real("2") times("*") variable("q") rparen(")") pow("^") real("2") divides("/") lparen("(") real("1.5") pow("^") real("4") rparen(")") 
